functions/Get-DbaServerAudit.ps1

```diff
@@ -42,43 +38,40 @@ Returns all Security Audits for the local and sql2016 SQL Server instances
 #>
 	[CmdletBinding()]
 	Param (
-		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
-		[DbaInstanceParameter]$SqlInstance,
+		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $true)]
+		[DbaInstanceParameter[]]$SqlInstance,
 		[PSCredential][System.Management.Automation.CredentialAttribute()]$Credential,
 		[switch]$Silent
 	)
 	
-	process
-	{
+	process {
 		foreach ($instance in $SqlInstance)
 		{
-			Write-Verbose "Attempting to connect to $instance"
-			try
-			{
+			try {
+				Write-Message -Level Verbose -Message "Connecting to $instance"
 				$server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $Credential
```

Comment:
This can be updated to the following to note minimum version of SQL Server it supports:

```
$server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $Credential -MinimumVersion 10
```

This now handles the warning message and outputs as such:
```
WARNING: [Get-DbaServerAudit][09:40:08] Failure | SQL Server version 10 required - [$instance] not supported.
```